test/invisible_ink_test.rb

```diff
@@ -29,8 +30,8 @@ def test_it_can_read_file
   end
 
   def test_it_can_create_a_new_file
-    assert_equal 2, @invisible_ink.write_file
-    assert_equal "Hi", File.read(@invisible_ink.new_file)
+    assert_equal 15, @invisible_ink.write_file
```

Comment:
This number kept changing, and 15 didn't make sense for "Hi".  But if it's 6 x 2 = 12 + 3 of those `\n` things, then it checks out.